doc/adr/release.md

```diff
@@ -0,0 +1,1019 @@
+# Release tool
+
+Create an executable that provides high-level and content-focused controls for
+creating releases from repositories using IOGX.
+
+## Motivation
+
+For engineers focused on the development and maintenance of a product, releases
+are only a means to distribute their works in a way that they are strongly
+associated with an identifier that is fit uses when communicating with users
```

Comment:
that is fit uses?

Reply:
https://github.com/input-output-hk/iogx/pull/29/commits/27fe9900bf1f06011e016c1e0d98d8b9d4f4b7da, better?